Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>*.iml\r\n.gradle\r\n/local.properties\r\n/.idea/caches\r\n/.idea/libraries\r\n/.idea/modules.xml\r\n/.idea/workspace.xml\r\n/.idea/navEditor.xml\r\n/.idea/assetWizardSettings.xml\r\n.DS_Store\r\n/build\r\n/captures\r\n.externalNativeBuild\r\n.cxx\r\nlocal.properties\r\n
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision cd94d9aa58f8c02c230cfe0b74a88b1c9ad5df6e)
+++ b/.gitignore	(date 1749158184645)
@@ -13,3 +13,4 @@
 .externalNativeBuild
 .cxx
 local.properties
+/app/
